src/js/components/AnalysisPanel/AnalysisFormElements/AnalysisMultiDatePicker.js

```diff
@@ -0,0 +1,135 @@
+import moment from 'moment';
+import DatePicker from 'react-datepicker';
+
+
+
+export default class AnalysisRangeSlider extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      selectedStartDate: moment(props.defaultStartDate, 'YYYY-MM-DD') || moment(new Date()),
+      selectedEndDate: moment(props.defaultEndDate, 'YYYY-MM-DD') || moment(new Date()),
+    };
+  }
+
+  componentDidMount() {
+    const { selectedStartDate, selectedEndDate } = this.state;
+    const {
+      calendarCallback,
+      analysisId,
+      startParamName,
+      endParamName,
+      multi,
+      valueSeparator,
+      combineParams
+    } = this.props;
+
+    calendarCallback(
+      selectedStartDate.format('YYYY-MM-DD'),
+      selectedEndDate.format('YYYY-MM-DD'),
+      analysisId,
+      combineParams,
+      multi,
+      startParamName,
+      endParamName,
+      valueSeparator,
+    );
+  }
+
+  componentDidUpdate(prevProps, prevState) {
+    const { selectedStartDate, selectedEndDate } = this.state;
+    const {
+      calendarCallback,
+      analysisId,
+      startParamName,
+      endParamName,
+      multi,
+      valueSeparator,
+      combineParams
+    } = this.props;
+
+    if (prevState.selectedStartDate !== selectedStartDate || prevState.selectedEndDate !== selectedEndDate) {
+      calendarCallback(
+        selectedStartDate.format('YYYY-MM-DD'),
+        selectedEndDate.format('YYYY-MM-DD'),
+        analysisId,
+        combineParams,
+        multi,
+        startParamName,
+        endParamName,
+        valueSeparator,
+      );
+    }
+  }
+
+  handleStartChange = date => {
+    this.setState({
+      selectedStartDate: date,
+    });
+  }
+
+  handleEndChange = date => {
+    this.setState({
+      selectedEndDate: date,
+    });
+  }
+
+  render() {
+    const { label, minDate, maxDate } = this.props;
+    const { selectedStartDate, selectedEndDate } = this.state;
+
+    return (
+      <div className='analysis-results__select-form-item-container'>
+        <div className='select-form-item-label'>{label}</div>
+        <DatePicker
+          customInput={<StartButton />}
+          showMonthDropdown
+          showYearDropdown
+          dropdownMode="select"
+          todayButton='Jump to today'
+          minDate={moment(minDate)}
+          maxDate={selectedEndDate}
+          selected={selectedStartDate}
+          onChange={this.handleStartChange}
+        />
+        <DatePicker
+          customInput={<EndButton />}
+          showMonthDropdown
+          showYearDropdown
+          dropdownMode="select"
+          todayButton='Jump to today'
+          minDate={selectedStartDate}
+          maxDate={moment(maxDate || new Date())}
+          selected={selectedEndDate}
+          onChange={this.handleEndChange}
+        />
+      </div>
+    );
+  }
+}
+
+const StartButton = ({ onClick, value }) => (
+  <div>
+    <label className='analysis-datepicker-button-label'>Start: </label>
```

Comment:
Eventually we want this stuff to reflect our current language! Maybe not for this PR specifically, but we should run a quick assessment of where we need to config out hard-coded text.